src/index.ts

```diff
@@ -18,8 +15,8 @@ const OptionDefaults: IOptions = {
   replaceFences: true,
   rolesAfter: "inline",
   directivesAfter: "block",
-  directives: { ...admonitions, ...images, ...code, ...tables },
-  roles: { ...roles }
+  directives,
```

Comment:
maybe the eventual behaviour here should be that users of the package that want to add extra roles/directives should be able to do it without having to redefine the entire mapping, i.e. the defaults are always used but then more can be added.

obviously sphinx has the concept of extensions: https://www.sphinx-doc.org/en/master/extdev/index.html#dev-extensions, whereby they "programatically" add individual roles/directives to the existing application:

```python
def setup(app):
   app.add_directive("name", klass)
   app.add_roles("name", callable)
```

Reply:
Fair, the way I was using it was:

```typescript
import { directivesDefault, Directive } from `markdown-it-docutils`

const opts = {
  directives: {...directivesDefault, cool: coolNewDirective}
}
```

If there is a class that eventually helps in the setup, then I think that makes sense as well down the road.

Reply:
Kicking off discussion in #17.